mantidimaging/core/io/loader/loader.py

```diff
@@ -94,12 +94,16 @@ def read_image_dimensions(file_path: Path) -> tuple[int, int]:
     return img.shape
 
 
-def load_log(log_file: Path) -> InstrumentLog:
+def load_log(log_file: Path | None) -> InstrumentLog:
+    if log_file is None:
+        return InstrumentLog([], Path(""))
```

Comment:
Do we need these `is None` cases here and in `load_shutter_counts`?

The functions that call it just take `Path`, so there should not be a case where it can be `None`